.github/workflows/testpypi.yml

```diff
@@ -21,27 +21,5 @@ jobs:
       with:
         name: python-package-distributions
         path: dist/        
-
-  publish-to-testpypi:
-    name: Publish Python 🐍 distribution 📦 to TestPyPI
-    needs:
-    - build
-    runs-on: ubuntu-latest
-
-    environment:
-      name: testpypi
-      url: https://test.pypi.org/p/acoular
-
-    permissions:
-      id-token: write  # IMPORTANT: mandatory for trusted publishing
-
-    steps:
-    - name: Download all the dists
-      uses: actions/download-artifact@v4
-      with:
-        name: python-package-distributions
-        path: dist/
     - name: Publish distribution 📦 to TestPyPI
-      uses: pypa/gh-action-pypi-publish@release/v1
-      with:
-        repository-url: https://test.pypi.org/legacy/          
\ No newline at end of file
+      run: hatch publish -r test -u __token__ -a ${{ secrets.HATCH_PUBLISH_TESTPYPI_API_TOKEN }}
```

Comment:
where is the test repo defined? shouldn't there be an entry in `pyproject.toml`?

Reply:
It has nothing to do with the pyproject.toml but with the repository (TestPyPI in this case)